site/training/training-overview.qmd

```diff
@@ -0,0 +1,140 @@
+:::
+
+### Sample models
+
+Sample models registered in the model inventory include:
+
+- Housing prices prediction model
+- Credit risk scorecard model
+- Customer churn prediction model
+- Interest rate time series forecasting model
+- Large language model (LLM) demo application
+
+Please note that we make updates to the models and datasets available in the training environment from time to time to provide you with our latest features. 
+
+### Sample notebooks 
+
+Each interactive notebook includes the sample code needed to automatically document a model: 
+
+Notebook | Inventory model |
+----------|-----------------|
+| **[Quickstart for model documentation](https://jupyterhub.validmind.ai/hub/user-redirect/lab/tree/quickstart_customer_churn_full_suite.ipynb)**<br>Gets you started with the basic process of documenting models with ValidMind, from the {{< var vm_dev >}} to the {{< var vm_ui >}}. | [Demo] Customer Churn Model
+| **[ValidMind introduction for model developers](https://jupyterhub.validmind.ai/hub/user-redirect/lab/tree/tutorials/intro_for_model_developers.ipynb)**<br>As a model developer, learn how the end-to-end documentation process works based on common scenarios you encounter in model development settings. | [Demo] Customer Churn Model
+| **[Document an application scorecard model](https://jupyterhub.validmind.ai/hub/user-redirect/lab/tree/code_samples/credit_risk/application_scorecard_demo.ipynb)**<br>Guides you through building and documenting an application scorecard model using the Lending Club sample dataset from Kaggle. | [Demo] Credit Risk Model
+| **[Prompt validation for large language models (LLMs)](https://jupyterhub.validmind.ai/hub/user-redirect/lab/tree/code_samples/nlp_and_llm/foundation_models_integration_demo.ipynb)**<br>Run and document prompt validation tests for a large language model (LLM) specialized in sentiment analysis for financial news. | [Demo] Foundation Model - Text Sentiment Analysis
+<!--- | Yes<br> | N/A | Credit Risk Model --->
+
+## Getting started
+
+What you try out first in the training environment depends on your interests: 
+
+::: {.grid}
+
+::: {.g-col-6}
+
+### Automated model testing & documentation 
+
+- Explore the sample notebooks
+- Run tests and test suites
+- Generate model documentation
+
+<form method="get" action="https://jupyterhub.validmind.ai/" target="_blank">
+   <button type="submit" style="color: white; background-color: #de257e; border-radius: 8px; border: none; font-size: 16px; padding: 6.25px 12.5px; margin-left: 16px; margin-bottom: 20px;">ValidMind Jupyter Hub</button>
+</form>
+
+<!---Also try out our cloud-based {{< var vm_ui >}} to get your model documentation ready for validation.--->
+:::
+
+::: {.g-col-6}
+
+### Model risk management & governance
```

Comment:
As mentioned elsewhere, one thing missing still is a repeatable, consistent training environment that we can periodically wipe or update. It's coming, along with a dedicated training organization on ValidMind, it's just not quite ready yet. 

Andres and I have discussed the requirements several times and it will include standard registered models, model documentation and validation reports already populated or in progress, and meaningful findings, reports, etc. That will also allow much better linking from some of the `TRY IT` pages. Stay tuned!